### PR TITLE
Add a Test_MDBT_nRF52833 Board

### PR DESCRIPTION
**Pull request by MWLEEmwlee**
Test

### CODE REVIEW

**Comment by carlescufi** [comment]
@MWLEEmwlee thanks for the PR!
Please take a look at https://docs.zephyrproject.org/latest/contribute/guidelines.html#pull-requests-and-issues to correctly format your commits.

Please do not close this PR, and instead use `git push -f` to fix the commits.

File: boards/arm/nrf52833dk_nrf52833_mdbt/Kconfig.board [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_NRF52833DK_NRF52833_MDBT
```
Comment (jfischer-no) [inline boards/arm/nrf52833dk_nrf52833_mdbt/Kconfig.board:4]: If it is about MDBT50Q-DB-33 board then it should be named MDBT50Q-DB-33, `config BOARD_MDBT50Q_DB_33`

File: tests/drivers/adc/adc_api/src/test_adc.c.bak [hunk 2]
```diff
@@ -0,0 +1,788 @@
+/*
```
Comment (jfischer-no) [inline tests/drivers/adc/adc_api/src/test_adc.c.bak:1]: You do not want to include this file.

File: boards/arm/nrf52833dk_nrf52833_mdbt/nrf52833dk_nrf52833_mdbt.dts [hunk 3]
```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2022 Nordic Semiconductor ASA
+// SPDX-License-Identifier: Apache-2.0
+
+/dts-v1/;
+#include <nordic/nrf52833_qiaa.dtsi>
+
+/ {
+	model = "NRF52833DK_NRF52833_MDBT";
+	compatible = "raytac,nrf52833dk-nrf52833-mdbt";
+
```
Comment (nordicjm) [inline boards/arm/nrf52833dk_nrf52833_mdbt/nrf52833dk_nrf52833_mdbt.dts:10]: Please name this board appropriately, if the board is called e.g. `MDBT50Q` then the board name and folder name should be `mdbt50q`

File: boards/arm/nrf52833dk_nrf52833_mdbt/nrf52833dk_nrf52833_mdbt.dts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+			reg = <0xc000 0x32000>;
+		};
+		slot1_partition: partition@3e000 {
+			label = "image-1";
+			reg = <0x3e000 0x32000>;
+		};
+		scratch_partition: partition@70000 {
+			label = "image-scratch";
+			reg = <0x70000 0xa000>;
+		};
```
Comment (nordicjm) [inline boards/arm/nrf52833dk_nrf52833_mdbt/nrf52833dk_nrf52833_mdbt.dts:39]: Do not include scratch partitions

**Review by nordicjm — CHANGES_REQUESTED** [review]

**Pull request closed without merging**